pyautosplit/process.py

```diff
@@ -10,8 +11,11 @@ def __init__(self, command, cwd):
                                         stdout=subprocess.DEVNULL,
                                         cwd=cwd)
 
+        time.sleep(1)
+
         self.debugger = ptrace.debugger.PtraceDebugger()
         self.dprocess = self.debugger.addProcess(self.process.pid, False)
+        self.base_address = int(self.dprocess.readMappings()[0].start)
```

Comment:
Why include this? This is the first address of the text segment, that is always located at 0x400000.

Reply:
I've made the assumption that `[0]` is always the current running program but I may got tricked by that and I have to look at it again because
This is not always at the same location and changes with every program startup. This will give a static starting point which offsets can rely on across startups.

Reply:
Addresses are always relative to the start of the program memory, so no need to do that.

Reply:
And [0] is the first segment of the process. You can look at the data yourself by reading `/proc/<pid>/maps`
[0] refers to the first segment (The text segment).

Reply:
> Addresses are always relative to the start of the program memory, so no need to do that.

I've also thought that this might be the case but I couldn't get it working. That's why this PR exists in the first place.